DockeriseRmap/Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM rocker/r-base:4.2.2
+
+USER  root
+
+RUN apt-get update -y && apt-get install -y libcurl4-openssl-dev samtools gawk tabix bcftools curl --no-install-recommends && rm -rf /var/lib/apt/lists/*
+RUN R -e "install.packages(c('tidyr', 'dplyr', 'data.table', 'stringr', 'Rsamtools', 'argparser', 'BiocManager'), dependencies=TRUE, repos='http://cran.rstudio.com/')"
+RUN R -e "BiocManager::install(c('GenomicAlignments'))"
+COPY /DockeriseRmap/mapV3.r /mapV3.r
+COPY /DockeriseRmap/map.sh /map.sh
+COPY /DockeriseRmap/error_check_map.sh /error_check_map.sh
+
+RUN adduser --disabled-password --gecos '' ubuntu && chsh -s /bin/bash && mkdir -p /home/ubuntu
+
+RUN mkdir /workdir
+RUN chown ubuntu /workdir
+
+# Add test run
+# rm test/repeats.MAP.vcf.gz will fall if the RScript didn't succeed in creating that file
+RUN mkdir -p test \
```

Comment:
I may not have been clear.  It's fine to put these commands into a bash script and copy that into the image then execute, e.g.

```
COPY test_script.sh /tmp/.
RUN /tmp/test_script.sh
```